docs/integrations/index.md

```diff
@@ -5,7 +5,7 @@
 Flyte is designed to be highly extensible and can be customized in multiple ways.
 
 ```{note}
-Want to contribute an example? Check out the {doc}`Example Contribution Guide <contribute>`.
+Want to contribute an example? Check out the [Documentation contribution guide](https://docs.flyte.org/en/latest/flytesnacks/contribute.html).
```

Comment:
this will be considered an external link as far as sphinx is concerned, and will always go to the `latest` version of the docs, regardless of version, can we use a `ref` or preserve the correct `doc` directive?

Reply:
Good call, I'll use a `ref` for this, since that doc has been moved to the flyte repo as part of the docs overhaul.